backend/src/graphql/resolver/BalanceResolver.ts

```diff
@@ -5,18 +5,24 @@ import { Resolver, Query, Ctx, Authorized } from 'type-graphql'
 import { Balance } from '@model/Balance'
 import { calculateDecay } from '@/util/decay'
 import { RIGHTS } from '@/auth/RIGHTS'
-import { Transaction } from '@entity/Transaction'
+import { Transaction, Transaction as dbTransaction } from '@entity/Transaction'
 import Decimal from 'decimal.js-light'
+import { GdtResolver } from './GdtResolver'
+import { TransactionLink as dbTransactionLink } from '@entity/TransactionLink'
+import { MoreThan, getCustomRepository } from '@dbTools/typeorm'
+import { TransactionLinkRepository } from '@repository/TransactionLink'
 
 @Resolver()
 export class BalanceResolver {
   @Authorized([RIGHTS.BALANCE])
   @Query(() => Balance)
   async balance(@Ctx() context: any): Promise<Balance> {
-    // load user and balance
     const { user } = context
     const now = new Date()
 
+    const gdtResolver = new GdtResolver()
+    const balanceGDT = await gdtResolver.gdtSum(context)
```

Comment:
this implies to have a successful call on `balance` you need the right `BALANCE` and `GDT_SUM` ?!

Reply:
No. gdtBlanace returns null when API call is not seccessful